src/backend.py

```diff
@@ -19,16 +27,18 @@ def __init__(self, http_client):
                                                      headers=data)
 
         response = await response.json()
+        skus = await self.get_skus()
 
         owned_products = []
         if 'products' in response:
             for platforms in response['products']:
                 for platform in platforms:
                     for game in platforms[platform]:
                         log.info(game)
-                        if game['sku'] and game['title'] and game['product_type'] and "paradox builds" in game['platforms']:

```

Comment:
Does removing the check for `"paradox builds"` cause any issues? In my testing, checking for the launcher on the next line was sufficient, but maybe it will be a problem for others?

Reply:
I've lost one game displayed  which is available in pdx client if we use 'paradoxLauncher' in skus check instead. 

Proposition
```
                        if game['sku'] and game['title'] and game['product_type']:
                            if (game['sku'] in skus and 'paradoxLauncher' in skus[game['sku']]['platform']) \
                                    or "paradox builds" in game['platforms']:
                                owned_products.append({'sku': game['sku'],
                                                       'title': game['title'],
                                                       'type': game['product_type']})
```

this way my game list is same as before and it contains your fix

Reply:
That works for me! I posted a new commit with the suggested changes.

Reply:
Great! I'm merging